### PR TITLE
migration: update README

### PR DESCRIPTION
**Pull request by Ptival**
Mostly because the note at the top was outdated, but I also changed a couple things.

### CODE REVIEW

File: migration/README.md [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
 
-To set up, we suggest using Python's virtualenv.  A possible setup is to run:
+To set up, we suggest using Python's virtualenv.  A possible setup is to run
+(replacing `<env>` with the name of the folder you want your virtual environment
+saved in):
 
 ```
-python3 -m venv env       # Creates a virtual environment in the 'env' directory
-source env/bin/activate   # Sets up the virtual environment in the **current** shell
+python3 -m venv <env>       # Creates a virtual environment in the 'env' directory
```
Comment (glguy) [inline migration/README.md:21]: I think it makes sense to be general here, but that stops people from copy/pasting as easily and it seems quite rare that people will need multiple virtual environments. I'm OK to leave it like this if you prefer it though.

File: migration/README.md [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+source <env>/bin/activate   # Sets up the virtual environment in the **current** shell
 ```
 
+The rest of the procedure uses `pip`.  You can choose to install it globally
```
Comment (glguy) [inline migration/README.md:25]: Once we're in a venv doesn't `pip3` come from the venv? Is there any point to using the `python3 -m pip` invocation?

File: migration/README.md [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
+
 In a shell where the environment is activated, you ought to be able to:
 
 ```
-python3 -m pip install -r requirements.txt
-# or, if you intend to modify the tool:
-python3 -m pip install -r requirements.txt -r requirements-dev.txt
+pip install -r requirements.txt
+# or, if you intend to work on the tool:
+pip install -r requirements.txt -r requirements-dev.txt
```
Comment (glguy) [inline migration/README.md:35]: For working on the tool we might even recommend installing it with `-e`

**Review by glguy — APPROVED** [review]

File: migration/README.md (hunk 1 above)
Comment (Ptival) [reply migration/README.md:21]: That makes sense, I think the comment makes it explicit enough that `env` is something under your control.

File: migration/README.md (hunk 3 above)
Comment (Ptival) [reply migration/README.md:35]: I did not know about `-e`, sounds neat!

File: migration/README.md (hunk 2 above)
Comment (Ptival) [reply migration/README.md:25]: Indeed that's the case, I did not realize, will simplify explanations!

**Pull request merged**